frontend/pages/sites/SiteListItem.jsx

```diff
@@ -86,11 +63,6 @@ SiteListItem.propTypes = {
     viewLink: PropTypes.string,
```

Comment:
i'm not sure about `viewLink` -- i wonder if it was what the link used to link to before we decided to make it the build history?